config/config.ini

```diff
@@ -1,5 +1,5 @@
 [database]
-dbname = flood_forecaster
```

Comment:
So we use the same db as them?

Reply:
I guess it's the schema that changes, should we put that in the conf as well?

Reply:
Correct, we use the same db, but a different schema, so we could access other schemas/tables/views, if needed.
I thought about adding it to the conf, but maybe we will create & use differrent schemas, so I left it out, as parameter for now.